src/pages/trader-tools/pnl-for-multipliers/_pnl-for-multipliers.js

```diff
@@ -144,7 +144,6 @@ const PnlMultipliersCalculator = () => {
                                 enableReinitialize
                                 initialValues={{
                                     direction: 'Up',
-                                    pnlMarginSymbol: 'USD',
```

Comment:
how about passing an empty string for the `pnlMarginSymbol` instead of removing it?

Reply:
> `MarginSymbol` instead of removing it?

Hi @mahdiyeh-fs May I know what is the advantage to pass empty string instead of removing? Cause it won’t be needed anymore hence I removed totally  🙏 🌹

Reply:
yeah, you're right. This value is not used anymore. good job +1